wp-content/mu-plugins/stat-collector/StatCollectorFunctions.php

```diff
@@ -61,8 +52,6 @@ function peu_data($staff, $client, $uid) {
     return;
   }
   $db = _get_db();
-  $db->suppress_errors();
-  $db->show_errors();
 
   if (! empty($staff)) {
     $result = $db->query($db->prepare("
```

Comment:
Starting at line 90, same thing. `data` doesn't exst, so you need to update the `error_log` to reflect the correct arguments.